src/components/dls/Skeleton/Skeleton.module.scss

```diff
@@ -47,6 +38,25 @@ $animationDuration: 8s;
   border-radius: var(--border-radius-circle);
 }
 
-.squared::before {
-  border-radius: var(--border-radius-sharp);
+@-webkit-keyframes loading {
+  0% {
+    opacity: 1;
+  }
+  50% {
+    opacity: 0.4;
+  }
+  100% {
+    opacity: 1;
+  }
+}
```

Comment:
I think the postcss in nextjs will handle this for us? or

Reply:
yes, I think it should. will remove it.